django/template/base.py

```diff
@@ -151,6 +151,9 @@ def __eq__(self, other):
             self.loader == other.loader
         )
 
+    def __ne__(self, other):
```

Comment:
Then we could slap an `if six.PY2` around it, what do you think? That way we'll know to remove it when we drop Python 2 support.